mustang/Cargo.toml

```diff
@@ -20,7 +20,7 @@ cc = { version = "1.0.68", optional = true }
 # TODO: Eventually, we should propose a `fde-phdr-rustix` backend option to
 # upstream `unwinding` so that it doesn't need to go through `dl_iterate_phdr`,
 # but `fde-phdr-dl` works for now.
-[target.'cfg(not(any(target_arch = "aarch64", target_arch = "arm", target_arch = "riscv64")))'.dependencies.unwinding]
+[target.'cfg(not(target_arch = "arm"))'.dependencies.unwinding]
 version = "0.1.1"
```

Comment:
Is there a new version number we can reference here, to be sure we're pulling in the updated version?

Reply:
the version that updated gimli to the crates.io version is `0.1.3`, as can be seen in https://github.com/nbdd0121/unwinding/commit/52086c73f88e2e24e146759191095d3281a02dc1